### PR TITLE
Add public badge for get payment request summary endpoint

### PR DESCRIPTION
**Pull request by alexBinns**
This PR adds the public badge to get payment request summary endpoint

### CODE REVIEW

**Review by Neeraj-Ghodla — APPROVED** [review]

**Comment by AndyClifford** [comment]
We should add this badge to the assets/asset transfer public summary endpoints

File: src/content/api/payment-requests.mdx [hunk 1]
```diff
@@ -576,7 +577,7 @@ Payment Activities are created when a Payment Request has been `created`, `paid`
   path="/api/payment-requests/{paymentRequestId}/summary"
   filename="payment-requests-get-summary"
 >
-  ## Get Payment Request Summary
+  ## Get Payment Request Summary <Badge text="public" />
```
Comment (AndyClifford) [inline src/content/api/payment-requests.mdx:580]: Was the design to use the default color?

Comment (alexBinns) [reply src/content/api/payment-requests.mdx:580]: Yea it was but @tom-blake didn't like adding the public badge as all the endpoints are public. There is a discussion in the design channel about alternative designs

**Comment by alexBinns** [comment]
This solution is not ideal and there is some more thought that needs to go into how we display these.

**Pull request closed without merging**